gradflow.html

```diff
@@ -164,6 +167,13 @@ <h1>Import Course</h1>
       </div>
     </div>
 
+    <div class="export-container">
+      <div class="dropdown">
+        <div class="dropdown-content" id="myDropdown">
+          <a href="javascript:void(0)" onclick="ExportPNG()">PNG</a>
+          <a href="javascript:void(0)" onclick="ExportJPEG()">JPEG</a>
+        </div>
+
```

Comment:
These divs aren't closed. Its breaking the HTML in subtle ways.

Reply:
thanks I did not catch that!